### ISSUE TITLE
Is it possible to add Horizontal interactive guideline in LineChart

### ISSUE THREAD

**Issue by StevenSeo** [novus/nvd3#1057]
Hi Guys, 

I want to add horizontal guideline in linechart. I've tried that in renderGuideLine func like below.  But it doesn't work. Any advices are very welcome. 

Thank you for sharing this amazing library.

            layer.renderGuideLine = function(x,y) {
                if (!showGuideLine) return;

				x = (x != null) ? nv.utils.NaNtoZero(x) : 0;
				y = (y != null) ? nv.utils.NaNtoZero(y) : 0;
				
				var crossPoint = [{"x1" : x, "x2" : x, "y1" : availableHeight, "y2" : 0}, {"x1" : availableWidth, "x2" : 0, "y1" : y, "y2" : y}];
				
                var line = wrap.select(".nv-interactiveGuideLine")
                    .selectAll("line")
		    .data(crossPoint);

                line.enter()
                    .append("line")
                    .attr("class", "nv-guideline")
                    .attr("x1", function(d) { return d.x1;})
                    .attr("x2", function(d) { return d.x2;})
                    .attr("y1", function(d) { return d.y1;})
                    .attr("y2", function(d) { return d.y2;})
                ;
                line.exit().remove();

            }

**Comment by robinfhu** [comment]
Hi, we don't have that feature.  I think it'll take quite a bit of work to actually implement it.  The renderGuideLine function can only handle vertical lines.  Sorry!

**Comment by StevenSeo** [comment]
Thank you for the reply. Can I have any feedback with the code that I made a change? Isn't it that easy that way?

**Comment by robinfhu** [comment]
Okay, so when you ran that code, did it generate two lines?  It looks like it should have, 

Also, what are you passing in as 'y' in the argument to the function?  It should equal the yScaled value of the data (something like yScale(yourYPoint))

**Comment by StevenSeo** [comment]
Thanks for the comment. I pass 'y' value correctly. But it doesn't show any lines. That's where I'm stuck now.

Before I call renderGuideLine func, I added below code for y value. I already confirmed the value is correct.

var pointYLocation;
if (typeof pointYLocation === 'undefined') pointYLocation = chart.yScale()(chart.y()(point,pointIndex));
....
interactiveLayer.renderGuideLine(pointXLocation,pointYLocation);
